packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -3423,36 +3514,17 @@ class _MenuPanelState extends State<_MenuPanel> {
         surfaceTintColor: surfaceTintColor,
         type: backgroundColor == null ? MaterialType.transparency : MaterialType.canvas,
         clipBehavior: widget.clipBehavior,
-        child: Padding(
-          padding: resolvedPadding,
-          child: ScrollConfiguration(
-            behavior: ScrollConfiguration.of(context).copyWith(
-              scrollbars: false,
-              overscroll: false,
-              physics: const ClampingScrollPhysics(),
-            ),
-            child: PrimaryScrollController(
-              controller: scrollController,
-              child: Scrollbar(
-                thumbVisibility: true,
-                child: SingleChildScrollView(
-                  controller: scrollController,
-                  scrollDirection: widget.orientation,
-                  child: Flex(
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    textDirection: Directionality.of(context),
-                    direction: widget.orientation,
-                    mainAxisSize: MainAxisSize.min,
-                    children: children,
-                  ),
-                ),
-              ),
-            ),
-          ),
-        ),
+        child: menuPanel,
       ),
     );
 
+    if (widget.menuOpacity != null) {
```

Comment:
Is there a special reason why `FadeTransition` must be outside while `SizeTransition` must be inside?